src/api/Places.ts

```diff
@@ -195,11 +198,11 @@ export default class Places extends API {
     )
   }
 
-  async getCategories() {
+  async getCategories(target: CategoryCountTarget = "all") {
```

Comment:
Maybe with this type `CategoryCountTarget = CategoryCountTargetOptions` we can do this, WDYT?

```suggestion
  async getCategories(target: CategoryCountTarget = CategoryCountTargetOptions.ALL) {
```